vaults-monitor/api/src/btc-vaults.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const { esploraClient } = require('esplora-client')
+const {
+  getBitcoinChainLastHeader,
+  getBitcoinCustodyAddress,
+  getBitcoinVaultStateAddress,
+  getPendingWithdrawalAmountSat,
+  getPendingWithdrawalCount,
+  getTunnelManagerStatus,
+  getVaultByIndex,
+  getVaultCounter,
+  getVaultStatus,
+  hemi,
+  hemiSepolia,
+} = require('hemi-viem')
+const viem = require('viem')
+
+function getHemiClient(chainId) {
+  const chain = { [hemi.id]: hemi, [hemiSepolia.id]: hemiSepolia }[chainId]
+  const transport = viem.http(chain.rpcUrls.default.http[0])
+  return viem.createPublicClient({ chain, transport })
+}
+
+async function getBitcoinChainHeight(network) {
+  const client = esploraClient({ network })
+  const height = await client.bitcoin.blocks.getBlocksTipHeight()
+  return { height }
+}
+
+async function getBitcoinChainData(client) {
+  const [bitcoin, bitcoinKit] = await Promise.all([
+    getBitcoinChainHeight(client.chain.testnet ? 'testnet' : 'mainnet'),
+    getBitcoinChainLastHeader(client),
+  ])
```

Comment:
```suggestion
  const [bitcoinHeight, bitcoinKit] = await Promise.all([
    getBitcoinChainHeight(client.chain.testnet ? 'testnet' : 'mainnet'),
    getBitcoinChainLastHeader(client),
  ])
```

Should this be `bitcoinHeight`? With the current naming, it sounds like the chain definition for Bitcoin 🤔

Reply:
Nevermind, I understood better what you wanted to achieve when reading

```js
 bitcoinChainData.bitcoin.height - bitcoinChainData.bitcoinKit.height
```